common/src/main/java/com/frostwire/bittorrent/BTEngine.java

```diff
@@ -329,6 +334,19 @@ public void download(TorrentCrawledSearchResult sr, File saveDir) {
         if (!exists) {
             File torrent = saveTorrent(ti);
             saveResumeTorrent(torrent);
+            if (ConfigurationManager.instance().getBoolean(Constants.PREF_KEY_TORRENT_DELETE_STARTED_TORRENT_FILES)) {
```

Comment:
`BTEngine` is in common, any android dependency here makes the desktop compilation to fail